features/step_definitions/common.ts

```diff
@@ -13,21 +14,43 @@ Given(
 );
 
 Given(
-  "The app {string} with {string} has no records",
-  function (appId: string, apiToken: string) {
-    const command = `record delete --app ${appId} --base-url $$TEST_KINTONE_BASE_URL --api-token ${apiToken} --yes`;
-    this.execCliKintoneSync(command);
-    if (this.response.status !== 0) {
-      throw new Error(`Resetting app failed. Error: \n${this.response.stderr}`);
-    }
+  "Load app ID of app {string} as env var: {string}",
+  function (appKey: string, destEnvVar: string) {
+    const credential = this.getCredentialByAppKey(appKey);
+    this.env = { [destEnvVar]: credential.appId, ...this.env };
   },
 );
 
+Given(
+  "Load app token of app {string} with exact permissions {string} as env var: {string}",
+  function (appKey: string, permission: string, destEnvVar: string) {
+    const permissions = permission
+      .split(",")
+      .map((p) => p.trim().toLowerCase()) as Permission[];
```

Comment:
I recommend adding validation that all the entries of `permissions` are compatible with `Permission` before type assertion.